draft-ietf-ppm-dap.md

```diff
@@ -1631,6 +1632,11 @@ This structure includes the following:
 
 * The number of reports included in the batch.
 
+* The smallest interval of time that contains the timestamps of all reports
+  included in the collection. Note that in the case of a `time_interval` type
```

Comment:
I think we should specify that the interval be aligned to `time_precision`, for clarity.
```suggestion
* The smallest interval of time that contains the timestamps of all reports
  included in the collection, such that the interval's start and duration are
  both multiples of the task's `time_precision` parameter. Note that in the case of a `time_interval` type
```

Reply:
Won't this be the case already if the client rounded the timestamp before upload? But then I suppose there's no harm in the leader doing that again just in case the client didn't for some reason.

Reply:
The chief ambiguity I want to squash here is that, if every timestamp is equal, we should be reporting only `Interval { ts, time_precision }`, not, say, `Interval { ts, 1 }`.